test/powershell/engine/Api/LanguagePrimitive.Tests.ps1

```diff
@@ -46,4 +46,60 @@ Describe "Language Primitive Tests" -Tags "CI" {
         [System.Management.Automation.LanguagePrimitives]::GetEnumerable($dt) | ForEach-Object { $count++ }
         $count | Should -Be 2
     }
+
+    It "TryCompare should succeed on int and string" {
+        $result = $null
+        [System.Management.Automation.LanguagePrimitives]::TryCompare(1, "1", [ref] $result) | Should -BeTrue
+        $result | Should -BeExactly 0
```

Comment:
We use `-BeExactly` for string only. Here we expect an integer and should use `-Be`.
Please review using `-BeExactly` below too.